IoTCoreDefaultApp/IoTCoreDefaultApp/Views/OOBEWelcome.xaml.cs

```diff
@@ -122,7 +122,7 @@ private void CancelButton_Clicked(object sender, RoutedEventArgs e)
                 // If the next screen is the main-page, navigate there, but also launch Cortana to its Consent Page independently
                 if (nextScreen == typeof(MainPage))
                 {
-                    CortanaHelper.LaunchCortanaToConsentPageAsyncIfNeeded();
+                    CortanaHelper.LaunchCortanaToConsentPageAsyncIfNeeded()?.Wait();
```

Comment:
Remove the **Wait** and put the **await** keyword at the beginning of the line. Will need to add the **async** keyword before between **private** and **void**.